portfolio/src/main/java/com/google/sps/servlets/HelloWorld.java

```diff
@@ -0,0 +1,29 @@
+package com.google.sps.servlets;
+import java.util.ArrayList;
+import com.google.gson.Gson;
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** Handles requests sent to the /hello URL. Try running a server and navigating to /hello! */
```

Comment:
This no longer handles requests sent to /hello so the comment should be updated. You can fix this later.